common/src/main/java/ac/grim/grimac/utils/nmsutil/MainSupportingBlockPosFinder.java

```diff
@@ -34,27 +32,33 @@ public MainSupportingBlockData findMainSupportingBlockPos(GrimPlayer player, Mai
         return new MainSupportingBlockData(null, true);
     }
 
+    private static class BestBlockHolder {
+        Vector3i pos = null;
```

Comment:
this is pointless but I don't think the compiler gets rid of it, extra jar size for no reason
```suggestion
        Vector3i pos;
```